style/Button.css

```diff
@@ -0,0 +1,82 @@
+.button-items{
+    padding:20px;
+    border: 0.5px solid rgba(128, 128, 128, 0.363);
+    margin: 20px;
+}
+.btn {
+    cursor: pointer;
+    padding: 12px;
+    min-width: 100px;
+    border:0px;
+  }
+  
+  .btn:focus, .btn.focus {
+    outline: 0;
+  }
+  
+  .btn-round-1 {
+    border-radius: 8px;
+  }
+
+  .btn-primary {
+    background: var(--blue-color);
+    color: var(--white-color);
+  }
+  
+  .btn-primary:hover {
+    background: #2980b9;
```

Comment:
use color variable

Reply:
used color variables in the website.Only for hover its not used